<?xml version="1.0"?>
<launch>
    <node name="avt_camera" pkg="avt_camera" type="avt_triggering" output="screen">
        <param name="~cam_IP" type="str" value="169.254.75.133" />
        <param name="~image_height" type="int" value="1200"/>
        <param name="~image_width" type="int" value="1600"/>
        <param name="~exposure_in_us" type="int" value="35000"/>
        <param name="~trigger_source" type="str" value="freerun"/>
        <param name="~frame_rate" type="double" value="55.0"/>
        <param name="~balance_white_auto" type="bool" value="false"/>
        <param name="~exposure_auto" type="bool" value="false"/>
        <param name="~gain" type="int" value="5"/>
    </node>
</launch>
